apiary.apib

```diff
@@ -2699,6 +2700,8 @@ Get a list of invoices.
             + updated_since: `2016-02-04T16:44:33+00:00` (string, optional)
             + purchase_order_number: `000023` (string, optional)
             + payment_reference: `+++084/2613/66074+++` (string, optional)
+            + invoice_date_after: `2020-01-01` (string, optional)  - This date is inclusive
+            + invoice_date_before: `2021-12-31` (string, optional)
```

Comment:
Maybe note here as well if it is inclusive or exclusive

Reply:
> Maybe note here as well if it is inclusive or exclusive

I've double checked and its actually also inclusive 🙈